src/modules/party/party.controller.test.ts

```diff
@@ -34,13 +42,22 @@ describe('PartyController', () => {
     partyServiceGetPartyIdentifierBySearchText = jest.fn();
     partyService.getPartyIdentifierBySearchText = partyServiceGetPartyIdentifierBySearchText;
 
-    controller = new PartyController(null, partyService);
+    partyExternalRatingServiceGetExternalRatingsForParty = jest.fn();
+    partyExternalRatingService.getExternalRatingsForParty = partyExternalRatingServiceGetExternalRatingsForParty;
+
+    partyExternalRatingServiceCreateExternalRatingForParty = jest.fn();
+    partyExternalRatingService.createExternalRatingForParty = partyExternalRatingServiceCreateExternalRatingForParty;
+
+    controller = new PartyController(partyService, partyExternalRatingService);
+
+    controllerSalesforceGetPartyByAlternateIdentifierPlaceholder = jest.fn();
+    controller.salesforceGetPartyByAlternateIdentifierPlaceholder = controllerSalesforceGetPartyByAlternateIdentifierPlaceholder;
```

Comment:
We should have a chat about this mocking :)

Reply:
As a result of our call, I created a mock which just returned the `CORPORATE` `assignedRatingCode`, and pulled this out into a separate class and mocked that